README.md

```diff
@@ -8,16 +8,45 @@ Welcome to the EOS.IO source code repository!
 The following instructions overview the process of getting the software, building it, and running a simple test network that produces blocks.
 
 ### Setting up a build/development environment
-This project is written primarily in C++14 and uses CMake as its build system. An up-to-date C++ toolchain (such as Clang or GCC) and the latest version of CMake is recommended. At the time of this writing, Nathan uses clang 4.0.0 and CMake 3.8.0.
+This project is written primarily in C++14 and uses CMake as its build system. An up-to-date C++ toolchain (such as Clang or GCC) and the latest version of CMake is recommended.
 
-### Installing Dependencies
-Eos has the following external dependencies, which must be installed on your system:
- - Boost 1.64
- - OpenSSL
- - LLVM 4.0 (Ubuntu users must install llvm-4.0 packages from https://apt.llvm.org/)
- - [secp256k1-zkp (Cryptonomex branch)](https://github.com/cryptonomex/secp256k1-zkp.git)
+Dependencies:
+* GCC 5.4.0 or Clang 4.0.0
```

Comment:
I will leave Clang 4.0.0 alone, there has been some problems with GCC 5.4.0 even if the project successfully complies

Reply:
Ok, I'll update accordingly.